<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clasificación</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Parametros</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QRadioButton" name="rbPiloto">
       <property name="text">
        <string>Mostrar por piloto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="rbConstructores">
       <property name="text">
        <string>Mostrar constructores</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="cbAnio"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Año</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="bBuscar">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>571</width>
     <height>411</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Posicion</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Puntos</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Victorias</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nacionalidad</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
